mbedtls/src/ssl/ciphersuites.rs

```diff
@@ -9,6 +9,7 @@
 use mbedtls_sys::types::raw_types::c_int;
 use mbedtls_sys::*;
 
+// Always use into() to convert to i32, do not use 'as i32'
```

Comment:
Is this intended as a doc comment? And why is not ok to type-cast? And conversion should happen to `c_int`, not `i32`.